### PR TITLE
Refactor: pop up saved message notif

### PR DESCRIPTION
**Pull request by rayfajars**
## Changes
- remove mixins from message notif
- refactor function pop up publish and saved message notif

## Evidence
title: Refactor pop up saved message notif
project: Jabar Super Apps
participants: @rayfajars @marsellavaleria19 @doohanas @Ibwedagama

### CODE REVIEW

**Review by doohanas — APPROVED** [review]

File: components/MessageNotif/Form/index.vue [hunk 1]
```diff
@@ -312,6 +329,7 @@ export default {
     },
     async validHandle (fileCorrect = true) {
       const isDataValid = await this.$refs.form.validate()
+      // TODO:
       // if (this.fieldMessageNotif.target.platform === '' && this.fieldMessageNotif.target.topic === '') {
       //   this.errMessageTarget = 'wajib memilih salah satu target.'
       // }
```
Comment (Ibwedagama) [inline components/MessageNotif/Form/index.vue:335]: helo @rayfajars is this comment used?

File: components/MessageNotif/Form/index.vue [hunk 2]
```diff
@@ -320,111 +338,88 @@ export default {
...
-        this.$router.push('/message-notif')
+    async emitOpenModalInformation (
+      modalNameEmitted,
+      isSuccessEmitted,
+      response
+    ) {
+      this.modalNameInformation = modalNameEmitted
+
+      if (modalNameEmitted === this.publishedConfirmationPopup.nameModal) {
+        const idMessage = await response.data.data.id
```
Comment (Ibwedagama) [inline components/MessageNotif/Form/index.vue:392]: hello @rayfajars, I just wondering if this `response.data.data.id` is a promise? if not maybe we don't have to use await 🤔

**Review by marsellavaleria19 — APPROVED** [review]

File: components/MessageNotif/Form/index.vue (hunk 1 above)
Comment (rayfajars) [reply components/MessageNotif/Form/index.vue:335]: yes bli @Ibwedagama is used, if API ready

File: components/MessageNotif/Form/index.vue (hunk 2 above)
Comment (rayfajars) [reply components/MessageNotif/Form/index.vue]: okey bli @Ibwedagama [fix here](https://github.com/jabardigitalservice/super-app-cms/pull/195/commits/36ae78e500cb33daae485eb1cc5b369bd28f1e34)

File: components/MessageNotif/Form/index.vue (hunk 1 above)
Comment (Ibwedagama) [reply components/MessageNotif/Form/index.vue:335]: oh okay maybe add a better TODO message @rayfajars 👍🏼

**Review by Ibwedagama — APPROVED** [review]

**Pull request merged**